packages/react/src/components/Theme/index.tsx

```diff
@@ -129,3 +133,7 @@ Theme.propTypes = {
 export function useTheme() {
   return React.useContext(ThemeContext);
 }
+
+export function usePrefersDarkScheme() {
```

Comment:
This also needs to be added to the top level exports https://github.com/carbon-design-system/carbon/blob/7e364d8c1d35cda0f92d95dc1abf3eef2546309d/packages/react/src/index.js#L303